unit-tests/game-model-tests.js

```diff
@@ -0,0 +1,44 @@
+(function(){
+    'use strict';
+    describe('*******TODO*******', function(){
+
+        var gameModel;
+
+        beforeEach(function(){
+            module('tombola.noughtAndCrosses.services');
+            inject(function($injector){
+                gameModel = $injector.get('gameModel')
+            });
+        });
+
+        it('Checks if a new game has been created',function(){
+            expect(gameModel.gameboard).to.equal('000000000');
+        });
+
+        it('player1 and player2 are set to human upon beginning a new game',function(){
+            expect(gameModel.player1).to.equal('human');
+            expect(gameModel.player2).to.equal('human');
+        });
+
+        it('nextPlayerType function cycles through the different player types',function(){
+            expect(gameModel.nextPlayerType()).to.equal('human');
```

Comment:
If the game model starts as Human, and you cycle shouldn't you end up with something else?
Which player are you cycling 1 or 2? 
If it isn't obvious from the test something is wrong - I suspect this method is internal to the game model & you should be testing something *like* gameModel.nextPlayer1Type() and similar methods for player 2. Additionally cycling one player isn't a full test  - better to keep moving to next until you are back where you started.